reframe/schemas/config.json

```
@@ -625,6 +626,7 @@
         "systems/partitions/time_limit": null,
         "systems/partitions/devices": [],
         "systems/partitions/extras": {},
+        "systems*/sched_options/access_on_submission_command": false,
```

I would suggest a shorter name:

```suggestion
        "systems*/sched_options/sched_access_in_submit": false,
```